src/PotentialCalculation/SuccessiveOverRelaxation/SOR.jl

```diff
@@ -71,33 +72,55 @@ Thus, there cannot be local extrema.
 """
 @inline @fastmath function handle_depletion(
     new_potential::T, 
-    point_type::PointType, 
+    imp_scale::T,
     neighbor_potentials::NTuple{6,T}, 
     q_eff_imp::T, 
     volume_weight::T,
     sor_const::T
-)::Tuple{T, PointType} where {T}
+)::Tuple{T, T} where {T}
     vmin::T = min(neighbor_potentials...)
     vmax::T = max(neighbor_potentials...)
```

Comment:
Not sure if this is faster but there is the function `minmax`:
```julia
vmin::T, vmax::T = minmax(neighbor_potentials...)
```
Also, is there a reason not to use `maximum` and `minimum` (or `extremum`) without `...`?
```julia
vmin::T , vmax::T = extremum(neighbor_potentials)
```

Reply:
I just checked: for "only" 6 values in `neighbor_potentials` it makes no difference, but `extremum` is better if its 40 values:

```julia
t = Tuple(randn(6))
# (-0.9443487734649653, -0.37544349629655793, -1.6098211403752445, -0.7270684898446809, 1.941463962273232, -0.9060439688986951)

determine_max(t) = max(t...)
@btime determine_max($t);

determine_maximum(t) = maximum(t)
@btime determine_maximum($t);
```
```
  9.698 ns (0 allocations: 0 bytes)
  9.686 ns (0 allocations: 0 bytes)
```

I just checked: for "only" 6 values in `neighbor_potentials` it makes no difference, but `extremum` is better if its 40 values:

```julia
t = Tuple(randn(40))

determine_max(t) = max(t...)
@btime determine_max($t);

determine_maximum(t) = maximum(t)
@btime determine_maximum($t);
```
```
  3.079 μs (82 allocations: 1.88 KiB)
  1.254 μs (42 allocations: 992 bytes)